### ISSUE TITLE
Metadata in stream flashls

### ISSUE THREAD

**Issue by BoweryCZ** [mangui/flashls#551]
We use HTML5 version floplayer (EngineName: hlsjs) where we read metadata from the stream.

`flowplayer(container, {
clip: {
live: true,
              sources: [
              {
                              type: "application/x-mpegurl",
                            src: "http://example.com/1-1/playlist.m3u8" 
              }],
hlsjs: {
                                             listeners: ["hlsFragParsingMetadata"]
                             },
}
})
.on("hlsFragParsingMetadata", function (e, api, data) {
        var str = (new TextDecoder("utf-8").decode(data.samples[0].data));
        console.log('FRAG_PARSING_METADATA: ' + str);
})`

The problem is Win7 IE11 where does not function EngineName: hlsjs but only EngineName: Flash.

Is it possible to set the flash plugin so as read metadata (ID3) using Javascript?

**Comment by phloxic** [comment]
Please make your case here: https://github.com/flowplayer/flowplayer/issues/830
Disclaimer: I have doubts whether it will be implemented for flash hls.

**Comment by joseluismg** [comment]
yes, it is possible to read the ID3 metadata events using javascript in the plugin flashls.

Just add the trigger  
`onNetStreamEvent : function (clip, type, info) {
     //You logic goes here
       }
`

**Comment by phloxic** [comment]
@joseluismg - yes with Flowplayer Flash, but @BoweryCZ is talking about the flash hls component for Flowplayer HTML5, if I understood correctly.
